service_capacity_modeling/models/common.py

```diff
@@ -118,14 +118,14 @@ def normalize_cores(
     return max(1, math.ceil(core_count / (target_speed / reference_speed)))
 
 
-def _headroom_approx(threads: int, thread_adj: float = 1.0) -> float:
+def _headroom_approx(cores: int, cores_boost: float = 1.0) -> float:
```

Comment:
Could we compromise on `cpu` to match the hardware language? These aren't cores (they are threads)